Automattic/jetpack
# E2E tests: only copy e2e workflow to relevant mirror repos

#### Changes proposed in this Pull Request:

Only copy the e2e GitHub workflow to mirror repositories that should trigger e2e tests.

#### Other information:


#### Jetpack product discussion
pd5faL-kN-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Tested locally by running build with `--for-mirrors`. `.github/files/gh-e2e` dir was only copied for the projects where `tests/e2e` path exists.
* Check [the build output](https://github.com/Automattic/jetpack/actions/runs/3515656978) for this PR. Projects with e2e tests should have the `e2e-tests.yml` file in `.github/workflows`
